packages/suite/src/support/messages.ts

```diff
@@ -9180,6 +9180,28 @@ export default defineMessages({
         id: 'TR_SOLANA_TX_CONFIRMATION_MAY_TAKE_UP_TO_1_MIN',
         defaultMessage: 'Confirmation of transaction may take up to <nowrap>1 minute</nowrap>',
     },
+    TR_SOLANA_TX_SEND_FAILED_TITLE: {
+        id: 'TR_SOLANA_TX_SEND_FAILED_TITLE',
+        defaultMessage: 'Send transaction failed',
+    },
+    TR_SOLANA_TX_SEND_FAILED_DESCRIPTION: {
+        id: 'TR_SOLANA_TX_SEND_FAILED_DESCRIPTION',
+        defaultMessage:
+            'The time to sign a Solana transaction is limited. It could no longer be submitted because it timed out and is no longer valid.',
+    },
+    TR_SOLANA_TX_CONFIRMATION_TIMER: {
+        id: 'TR_SOLANA_TX_CONFIRMATION_TIMER',
+        defaultMessage: '{remainingTime} seconds left',
+    },
+    TR_SOLANA_TX_CONFIRMATION_TIMER_SHORT: {
+        id: 'TR_SOLANA_TX_CONFIRMATION_TIMER_SHORT',
+        defaultMessage: '<strong>{remainingTime}s</strong> left to confirm',
+    },
+    TR_SOLANA_TX_CONFIRMATION_TIMER_DESCRIPTION: {
+        id: 'TR_SOLANA_TX_CONFIRMATION_TIMER_DESCRIPTION',
+        defaultMessage:
+            'Due to Solana constraints, you have ~1 minute to check everything before your transaction times out.',
```

Comment:
```suggestion
            'Due to Solana constraints, you have ~1 minute to check everything before your transaction times out.',
```

@vytick let's do it 45 seconds? There needs to be buffer for fetching hash, broadcast and confirmation

Reply:
or let's do 1 minute and speed up timer to 1 second === 0.75 second? 😀

Reply:
So? 😄